example-wasm/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@realm/example-wasm",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "wireit",
+    "build": "wireit",
+    "eject": "react-scripts eject"
+  },
+  "wireit": {
+    "start": {
+      "command": "craco start",
+      "dependencies": [
+        "../packages/realm:build:browser",
+        "../packages/realm:bundle",
+        "../packages/realm-react:bundle"
+      ]
+    },
+    "build": {
+      "command": "craco build",
+      "dependencies": [
+        "../packages/realm:build:browser",
+        "../packages/realm:bundle",
+        "../packages/realm-react:bundle"
+      ]
+    }
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "dependencies": {
+    "@craco/craco": "^7.1.0",
+    "@realm/react": "^0.4.3",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
```

Comment:
`@types`, `@testing-library`, `@craco`, `typescript` and `realm-scripts` can be moved to `devDependencies`.

Reply:
I'll go ahead and move these 👍 (it all ends up under regular deps by default with CRA)

Reply:
Done ✅

Reply:
Strange that CRA puts them in the deps. 🤔